doc/source/serve/core-apis.rst

```diff
@@ -248,6 +248,11 @@ code and set this number based on end to end latency objective.
   The ``version`` field is required for autoscaling. We are actively working on removing
   this limitation.
 
+.. note::
+  The Ray Serve Autoscaler is an application-level autoscaler that sits on top of the :ref:`Ray Autoscaler<cluster-index>`.  
+  Concretely, this means that the Ray Serve autoscaler asks Ray to start a number of actors, one for each replica, based on ``num_replicas``.
```

Comment:
I think the relationship between `num_replicas` and the autoscaler may not be  obvious to readers. Instead of saying "based on `num_replicas`," how about saying "based on request demand?"

Reply:
Oh yeah that's unclear, let me update it